src/windows.toit

```diff
@@ -143,26 +174,62 @@ is-absolute "../foo/bar"       // False.
 ```
 */
 is-absolute path/string -> bool:
+  return is-absolute-volume path or is-absolute-unc path
+
+/**
+Path starts with a UNC volume.
+For example: `//host/share`, `//./UNC/host/share`, ...
+*/
+is-absolute-unc path/string -> bool:
   volume-name-size := volume-name-size_ path
 
   if volume-name-size == 0:
     // Path doesn't have a volume name.
     // For example: `foo/bar`, `../foo`, `\foo\bar`, ...
     return false
 
-  assert: path.size >= 2
-  if is-separator path[0] and is-separator path[1]:
-    // Path starts with a UNC volume.
-    // For example: `//host/share`, `//./UNC/host/share`, ...
-    return true
+  return is-separator path[0]
 
-  if path.size == volume-name-size:
-    // Path is just a volume name.
-    // For example: `c:`.
-    return false
+/**
+Path starts with an absolute drive indicator.
+For example: `c:\\', 'd:/`, ...
+*/
+is-absolute-volume path/string -> bool:
```

Comment:
To me a UNC path is also a volume, and is always absolute.
So it feels weird that `//host/share/foo` says false to `is-absolute-volume`.

Reply:
What would you suggest the naming? An absolute path that starts with a driver letter: `is-absolute-path-starting-with-drive-letter'?

Reply:
I think `is-drive-absolute_` is much better. yes.